.github/workflows/nightly.yml

```diff
@@ -9,12 +9,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: thebritican/fetch-latest-release@v2.0.0
```

Comment:
This action does the trick: https://github.com/marketplace/actions/fetch-latest-release